SetupWizardApplication exception (cannot sign in to Google w/ WSA 1.8.32822.0)

I've installed `WSA 1.8.32822.0` with `open_gapps-x86_64-11.0-pico-20211113.zip` on my HP Envy Ryzen 4500U laptop.

Now when I click green `Sign In` button in Google Play nothing happens visually. In `logcat` I have the following error messages:

```
11-15 17:22:21.407  7010  7010 E .android.gms.u: Not starting debugger since process cannot load the jdwp agent.
11-15 17:22:21.610  7046  7046 E oid.setupwizar: Not starting debugger since process cannot load the jdwp agent.
11-15 17:22:21.675   215   293 E HalDevMgr: getAllChipInfo: called but mWifi is null!?
11-15 17:22:21.677   215   293 E HalDevMgr: getAllChipInfo: called but mWifi is null!?
11-15 17:22:21.680  7046  7046 E AndroidRuntime: FATAL EXCEPTION: main
11-15 17:22:21.680  7046  7046 E AndroidRuntime: Process: com.google.android.setupwizard, PID: 7046
11-15 17:22:21.680  7046  7046 E AndroidRuntime: java.lang.RuntimeException: Unable to create application com.google.android.setupwizard.SetupWizardApplication: java.lang.RuntimeException: java.lang.NoSuchFieldException: No field isCarrierAp in class Landroid/net/wifi/ScanResult; (declaration of 'android.net.wifi.ScanResult' appears in /apex/com.android.wifi/javalib/framework-wifi.jar)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6724)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.ActivityThread.access$1300(ActivityThread.java:237)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:223)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7664)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
11-15 17:22:21.680  7046  7046 E AndroidRuntime: Caused by: java.lang.RuntimeException: java.lang.NoSuchFieldException: No field isCarrierAp in class Landroid/net/wifi/ScanResult; (declaration of 'android.net.wifi.ScanResult' appears in /apex/com.android.wifi/javalib/framework-wifi.jar)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at mm.a(PG:318)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at lr.l(PG:292)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at lr.a(PG:206)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at lr.<init>(PG:112)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at mc.a(PG:141)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at mc.d(PG:50)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at mc.b(PG:79)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at ate.a(PG:36)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at aqz.a(PG:31)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at com.google.android.setupwizard.SetupWizardApplication.onCreate(PG:11)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1192)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6719)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        ... 8 more
11-15 17:22:21.680  7046  7046 E AndroidRuntime: Caused by: java.lang.NoSuchFieldException: No field isCarrierAp in class Landroid/net/wifi/ScanResult; (declaration of 'android.net.wifi.ScanResult' appears in /apex/com.android.wifi/javalib/framework-wifi.jar)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at java.lang.Class.getDeclaredField(Native Method)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        at mm.a(PG:316)
11-15 17:22:21.680  7046  7046 E AndroidRuntime:        ... 19 more
11-15 17:22:21.719  7010  7042 E goldfish-address-space: latte_get_buffer_fd failed to open shared memory region /mnt/shared_mem/gralloc_8573157376: Permission denied
11-15 17:22:21.719  7010  7042 E .android.gms.u: AddressSpaceStream::create failed (map ring storage)
11-15 17:22:21.719  7010  7042 E HostConnection: Failed to create AddressSpaceStream for host connection!!!
11-15 17:22:21.719  7010  7042 E EGL_emulation: Failed to establish connection with the host
11-15 17:22:21.886    57    57 E tombstoned: Tombstone written to: /data/tombstones/tombstone_30
11-15 17:22:21.938   215  2644 E EGL_emulation: eglCreateContext: 0x78088be0e490: maj 3 min 0 rcv 3
11-15 17:22:22.744   215   446 E InputDispatcher: But another display has a focused window:
11-15 17:22:22.744   215   446 E InputDispatcher: Display #3 has focused window: 'Window{186507b u0 com.android.vending/com.google.android.finsky.unauthenticated.activity.UnauthenticatedMainActivity}'
```

P. S. In Google Play I've clicked menu button and updated Contacts, YT Music and Photos from there. So it can successfully connect to Google servers.
UPD: the same issue as #123 #126 #127 #134

--- a/README.md
+++ b/README.md
@@ -239,4 +239,4 @@ You are now root.
 
 # Currently known issues
 
-WSA 1.8.32822.0 may potentially prevent users from signing in to the Google Play Store, reasons are under investigation. See details and [#128](https://github.com/WSA-Community/WSAGAScript/issues/128) for previous version.
+- [Issues](https://github.com/WSA-Community/WSAGAScript/issues)

--- a/apply.sh
+++ b/apply.sh
@@ -154,9 +154,13 @@ chcon --reference=$MountPointProduct/etc $MountPointProduct/build.prop
 chcon --reference=$MountPointVendor/etc $MountPointVendor/build.prop
 
 echo "Applying SELinux policy"
+SELinuxPolicy="(allow gmscore_app self (vsock_socket (read write create connect)))"
+SELinuxPolicy="${SELinuxPolicy}\n(allow gmscore_app device_config_runtime_native_boot_prop (file (read)))"
+SELinuxPolicy="${SELinuxPolicy}\n(allow gmscore_app system_server_tmpfs (dir (search)))"
+SELinuxPolicy="${SELinuxPolicy}\n(allow gmscore_app system_server_tmpfs (file (open)))"
 # Sed will remove the SELinux policy for plat_sepolicy.cil, preserve policy using cp
 cp $InstallDir/etc/selinux/plat_sepolicy.cil $InstallDir/etc/selinux/plat_sepolicy_new.cil
-sed -i 's/(allow gmscore_app self (process (ptrace)))/(allow gmscore_app self (process (ptrace)))\n(allow gmscore_app self (vsock_socket (read write create connect)))\n(allow gmscore_app device_config_runtime_native_boot_prop (file (read)))/g' $InstallDir/etc/selinux/plat_sepolicy_new.cil
+sed -i "s/(allow gmscore_app self (process (ptrace)))/(allow gmscore_app self (process (ptrace)))\n${SELinuxPolicy}/g" $InstallDir/etc/selinux/plat_sepolicy_new.cil
 cp $InstallDir/etc/selinux/plat_sepolicy_new.cil $InstallDir/etc/selinux/plat_sepolicy.cil
 rm $InstallDir/etc/selinux/plat_sepolicy_new.cil
 

